arcgis-ios-sdk-samples/Display information/Custom dictionary style/README.md

```diff
@@ -33,4 +33,4 @@ To learn more about how styles in dictionary renderers work, see the topic [Disp
 
 ## Tags
 
-dictionary, military, renderer, style, stylx, unique value, visualization
\ No newline at end of file
+dictionary, military, renderer, style, stylx, unique value, visualization
```

Comment:
Please add a new line after the tags if it is not already included in #886.

Reply:
There is already a newline here. A `no newline at end of file` symbol will be shown here if there is no newline. 

![image](https://user-images.githubusercontent.com/9660181/82712065-07bcb580-9c3c-11ea-80a1-6e388f4f6098.png)

I tend to keep `README.md` with a trailing newline and `README.metadata.json` without, as text files usually have that empty line at EOF, whereas most JSON encoder does not add newline at EOF. 

But it is totally optional to make this change and doesn't have effect on the final rendered output. 🙂